lib/github-client.js

```diff
@@ -174,6 +174,29 @@ GithubClient.prototype.collectReleasePRs = function (releasePR) {
   })
 }
 
+GithubClient.prototype.getLatestMergedReleasePR = function () {
+  var self = this
+
+  return self.get(self.pullRequestEndpoint(), {
+    base: 'release',
+    state: 'open',
```

Comment:
Is this correct? If you want the merged Release PR, shouldn't it be `state: 'merged'` or `state: 'closed'`?

Reply:
Release branch has `hotfix` PRs and Release PRs.

Okay to query `all` but remember you will need to check if it is a `release` PR type.

PRs can have `labels`, so you could also add some label to Release PRs and use that for filtering.